src/api/player.js

```diff
@@ -115,6 +121,16 @@ Player.prototype.getVolume = function() {
 };
 
 /**
+ * Sets the mute state of the video element. true is muted, false is unmuted.
+ */
+Player.prototype.mute = function(muteState) {
+  var data = {
+    muteState: muteState
+  };
```

Comment:
Yeah, sorry, you are right. I will probably need to change my volume impl to look more like what you did. Thanks!